nce/NCE-Pairwise-SM/main.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import random
+import logging
+
+import torch
+from torchtext import data
+
+from args import get_args
+from trec_dataset import TrecDataset
+from utils.relevancy_metrics import get_map_mrr
+
+from datasets.trecqa import TRECQA
+import os
```

Comment:
Group Python module together. According to [PEP 8](https://www.python.org/dev/peps/pep-0008/):

> Imports should be grouped in the following order:
> standard library imports
> related third party imports
> local application/library specific imports
> You should put a blank line between each group of imports.